### PR TITLE
Add Frontend View and Styles

### PR DESCRIPTION
**Pull request by CrocAx**
# PR summary #1
 I have started some nice design on the frontend and used Andrei's mock data to show how it should be presented.
 
 ### What I worked with:
 - Added all same routes: Dashboard, Tasks, Activities, Statistics
 - Used Andrei's mock data to represent how design would look like
 - Added nice transition from Landing page to Dashboard
 
 > Note: tried adding transitions between the routes. Going to fix that later.

### CODE REVIEW

**Review by Andebugulin — APPROVED** [review]
#[14645fa](https://github.com/martimolanes/TATASk_Manager/pull/5/commits/14645fadb845e5f56f058471b1db8e1310840bb2) LGTM! Nice job, keep it up, buddy.

File: client/package-lock.json
Comment (martimolanes) [inline client/package-lock.json:1]: this file probably is generated with a different package manager, like `npm`, we are using bun, so this file is not necessary.

Comment (martimolanes) [reply client/package-lock.json:1]: try deleting the file, also from the git control version and run `bun install` :)

**Review by martimolanes — CHANGES_REQUESTED** [review]
Okey, this is a very good effort and progress. Thank you @CrocAx for working on this. Even so I'm concerned of two major things:
1. This PR collides with the #4, so we need to close the other one and pick the changes we want of that PR, no?
2. Im not sure but I think, the architecture of the react router it can be better, because right now you reload the hole page, each time one of the navbar links are clicked. I think i would like to see a  more granular Router, with the navbar being a root element and the other pages being a children, like the [official example of react router]().
```jsx
const router = createBrowserRouter(
  createRoutesFromElements(
    <Route
      path="/"
      element={<Root />}
      loader={rootLoader}
      action={rootAction}
      errorElement={<ErrorPage />}
    >
      <Route errorElement={<ErrorPage />}>
        <Route index element={<Index />} />
        <Route
          path="contacts/:contactId"
          element={<Contact />}
          loader={contactLoader}
          action={contactAction}
        />
        <Route
          path="contacts/:contactId/edit"
          element={<EditContact />}
          loader={contactLoader}
          action={editAction}
        />
        <Route
          path="contacts/:contactId/destroy"
          action={destroyAction}
        />
      </Route>
    </Route>
  )
);
```
it would be cool to see your thoughts with some context of your decisions

File: client/package-lock.json
Comment (CrocAx) [reply client/package-lock.json:1]: ready

**Review by CrocAx — COMMENTED** [review]
i changed the things you requested, now working fine!

**Review by martimolanes — APPROVED** [review]
LGTM! im not merging it yet, im planning to review it deeply

**Comment by Andebugulin** [comment]
Great job @CrocAx 

#### I have noticed that the data is not being handled by context, that had been implemented already. 

So, I just added usage of context to our components: ```statistics.tsx, tasks.tsx, activities.tsx, DataContext.tsx```

**Review by martimolanes — APPROVED** [review]
The next time, better use another PR for this new changes, but its okey. lets merge it

**Pull request merged**